packaging/makeself/build.sh

```diff
@@ -6,7 +6,7 @@
 
 export NETDATA_BUILD_WITH_DEBUG=0
 
-while [ ! -z "${1}" ]
+while [ -n "${1}" ]
```

Comment:
Can we use the convention that we use elsewhere?

```suggestion
while [ -n "${1}" ]; do
```